src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.html

```diff
@@ -6,12 +6,10 @@
 -->
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column" (mouseover)="hover=true" (mouseleave)="hover=false">
 	<markdown-toolbar-component #markdownToolbar *ngIf="previewFeaturesEnabled === true && isEditMode" [cellModel]="cellModel"></markdown-toolbar-component>
-	<div class="notebook-text" style="flex: 0 0 auto;">
+	<div class="notebook-text" [class.edit-mode]="isEditMode">
 		<code-component *ngIf="isEditMode" [cellModel]="cellModel" (onContentChanged)="handleContentChanged()" [model]="model" [activeCellId]="activeCellId">
 		</code-component>
-	</div>
-	<div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: row">
-		<div #preview link-handler [isTrusted]="isTrusted" [notebookUri]="notebookUri" class="notebook-preview" style="flex: 1 1 auto">
+		<div #preview link-handler *ngIf="showPreview" [isTrusted]="isTrusted" [notebookUri]="notebookUri" class="notebook-preview">
```

Comment:
This is not set up properly in the component or component action:
*ngIf="showPreview"
I'm looking for a way to suppress rendering of this element, or hide it,  without producing a nativeElement error.

Reply:
I found that not rendering the preview component results in an element reference error in the textCell.component. If instead I add this, I can trigger show/hide with the addition of a class:
[class.hide-preview]="showPreview === false"